src/main/java/net/minestom/server/world/biomes/Biome.java

```diff
@@ -10,7 +11,7 @@
 import java.util.Locale;
 import java.util.concurrent.atomic.AtomicInteger;
 
-public final class Biome {
+public final class Biome implements ProtocolObject {
     public static final AtomicInteger ID_COUNTER = new AtomicInteger(0);
     private static final BiomeEffects DEFAULT_EFFECTS = BiomeEffects.builder()
```

Comment:
It doesn't make sense to hardcode plains while we are loading the rest. Also it would be nice generally to get plains from the data so this doesn't need to be updated if its ever changed.